pkg/motor/login.go

```diff
@@ -40,15 +43,38 @@ func (mtr *motorNodeImpl) Login(request mt.LoginRequest) (mt.LoginResponse, erro
 
 // LoginWithKeys creates a motor node from a LoginRequest
 func (mtr *motorNodeImpl) LoginWithKeys(request mt.LoginWithKeysRequest) (mt.LoginResponse, error) {
-	if request.Did == "" {
+	if request.AccountId == "" {
 		return mt.LoginResponse{}, fmt.Errorf("did must be provided")
 	}
 
-	mtr.Address = request.Did
+	// Create Client instance
+	mtr.Cosmos = client.NewClient(client.ConnEndpointType_BETA)
+
```

Comment:
I think webisoft built out a wrapper for this to grab the same one across the entire tool chain. Also shouldn’t this be wrapped in a singleton?

Reply:
Where is the wrapper?

Reply:
https://github.com/sonr-io/sonr/pull/636/files#r989477398

Reply:
This is where I thought it was used. Let me know if it is wrong and also if it should be added to the backlog

Reply:
Fixed